smartpy/optimizers/sgd.py

```diff
@@ -0,0 +1,51 @@
+import theano
+import theano.tensor as T
+from collections import OrderedDict
+
+import numpy as np
+from . import Optimizer
+
+
+class SGD(Optimizer):
+    def __init__(self, model, loss_fct, dataset, batch_size=1, update_rules=None, param_modifiers=None):
```

Comment:
I don't think this is necessary in the __init__ update_rules=None, param_modifiers=None